### PR TITLE
Added doxygen comments to player.h and player.c in score

### PR DESCRIPTION
**Pull request by ikbenhomer**

### CODE REVIEW

File: score/player.c [hunk 1]
```diff
@@ -10,22 +14,35 @@ void player_init(struct Player* player)
 
     player->highScore = 0;
 }
-
+/**
+ * @param currPlayer return values of player initial array
```
Comment (Kwask) [inline score/player.c]: This @param should describe what the `currPlayer` parameter is, then below that you can do @return to describe what the return value represents

File: score/player.c [hunk 2]
```diff
@@ -10,22 +14,35 @@ void player_init(struct Player* player)
...
+/**
+ * @param currPlayer return values of player initial array 
+ */
 char get_initial(struct Player currPlayer)
 {
     return currPlayer.initial;
 }
 
+/**
+ * @param currPlayer return value of player current score
```
Comment (Kwask) [inline score/player.c]: same here as above

File: score/player.c [hunk 3]
```diff
@@ -10,22 +14,35 @@ void player_init(struct Player* player)
...
+ */
+
 int get_current_score(struct Player currPlayer)
 {
     return currPlayer.currentScore;
 }
 
+/**
+ * @param currPlayer return value of player high score
```
Comment (Kwask) [inline score/player.c]: and also same here, @param for `currPlayer` isn't accurate, needs @return

**Review by Kwask — CHANGES_REQUESTED** [review]
Needs a few @return's and some @param's need to be changed in the doxygen comments.

**Review by brettsalyer — APPROVED** [review]

**Pull request merged**